Makefile

```diff
@@ -95,7 +96,7 @@ tree-status: ## Verify tree is clean and all changes are committed
 build-cron: build-pubsub build-bq-transfer build-github-server build-webhook build-add-script \
 	  build-validate-script build-update-script
 
-build-targets = generate-docs build-proto build-scorecard build-cron dockerbuild
+build-targets = generate-docs build-proto build-scorecard build-cron ko-build-binaries dockerbuild
```

Comment:
Does this need to be `ko-build-everything`?

Reply:
Thanks, @azeemshaikh38! 🤦